docs/public/setup.md

```diff
@@ -101,7 +101,7 @@ This procedure ensures you not only get the correct version of `invoke`, but als
 
 ### Golang
 
-You must [install Golang](https://golang.org/doc/install) version `1.23.3` or later. Make sure that `$GOPATH/bin` is in your `$PATH` otherwise `invoke` cannot use any additional tool it might need.
+You must [install Golang](https://golang.org/doc/install) version `1.23.5` or later. Make sure that `$GOPATH/bin` is in your `$PATH` otherwise `invoke` cannot use any additional tool it might need.
```

Comment:
```suggestion
You must [install Golang](https://golang.org/doc/install) version `1.23.5` or later. Make sure that `$GOPATH/bin` is in your `$PATH`, otherwise `invoke` cannot use any additional tool it might need.
```